jwql/website/apps/jwql/oauth.py

```diff
@@ -63,6 +63,8 @@ def register_oauth():
     """
 
     # Get configuration parameters
+    for key in 'client_id client_secret auth_mast'.split():
```

Comment:
That is an interesting way to iterate through a list! But I wonder if it would read easier if you just did:

```python
for key in ['client_id', 'client_secret', 'auth_mast']:
```